MKOB/kobwindow.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, root, VERSION):
         self.varCircuitCloser = tk.IntVar()
         chkCktClsr = tk.Checkbutton(lfm1, text='Circuit Closer',
                 variable=self.varCircuitCloser)
+        chkCktClsr.config(state='disabled')  # temporary
```

Comment:
I prefer a comment of:
 "# TODO: temporary". 
just because most development tools will highlight/list 'TODO' commented lines. The other option is that we add 'temporary' as a comment keyword for the tools to list.

An example is that I can bring up a list of all 'TODO' lines in my IDE (VS Code). We can also automatically generate a list of 'TODO' lines as part of a check-in if we enable some of the optional validation in GitHub (was waiting to do that until we are further along).

Reply:
As a note...
On the teams I worked on we used `TODO:` for something that should be addressed/done 'at some point' but didn't necessarily need to be done for the current release. We used `ZZZ:` for something we wanted to check in to be tested (for primary functionality) but needs to be addressed before being accepted for a release.

So, when doing a final check of code for a release, code with `TODO:` can be accepted, but code with `ZZZ:` needs to be addressed.

Reply:
I made this change. Please keep reminding me if I forget in the future.